docs/ENVIRONMENT.md

```diff
@@ -272,6 +272,15 @@ All tools:
   it is either set to `system` or `user` depending on whether the NSS/PAM
   module is called by systemd in `--system` or `--user` mode.
 
+* `$SYSTEMD_SUPPORT_DEVICE, SYSTEMD_SUPPORT_MOUNT, SYSTEMD_SUPPORT_SWAP` -
```

Comment:
Write as:

```
`$SYSTEMD_SUPPORT_DEVICE`, `$SYSTEMD_SUPPORT_MOUNT`, `$SYSTEMD_SUPPORT_SWAP` – …
```

i.e. an individual set of backticks, and prefixing `$` for each var